index.bs

```diff
@@ -474,8 +474,16 @@ An implementation may define <dfn export>extension modules</dfn>. These must hav
 [=module name=] that contains a single colon "<code>:</code>" character. The
 part before the colon is the prefix; this is typically the same for all
 [=extension modules=] specific to a given implementation and should be unique for a
-given implementation. Such modules extend the [=local end definition=] and [=remote end definition=]
-providing additional groups as choices for the defined [=commands=] and [=events=].
+given implementation.
+Such modules extend the [=local end definition=] and [=remote end definition=]
+providing additional groups as choices for the defined [=commands=], [=events=] and types.
+
+External specifications may define their own modules that extend the protocol.
+The recommendation is that such modules should have a [=module name=] that does
+not contain a colon "<code>:</code>" character, and that it is preferred
+to add new modules rather than extending existing ones.
+Such modules may extend the [=local end definition=] and [=remote end definition=] providing
+additional groups as choices for the defined [=commands=], [=events=] and types.
```

Comment:
```suggestion
Other specifications may define their own WebDriver-BiDi modules that define the protocol.
Such modules must not not have a name which contains a colon (<code>:</code>) character,
not must they define command names, event names, or property names that contain that character.

Authors of external specifications are encouraged to to add new modules rather than extending
existing ones. Where it is desired to extend an existing module, it is preferred to integrate the
extension directly into the specification containing the original module definition.
```

Or something.